README.md

```diff
@@ -16,9 +16,7 @@
        - [vcf2maf](#vcf2maf)
   - [Inputs](#inputs)
   - [Outputs](#outputs)
-  - [Testing and Validation](#testing-and-validation)
-    - [Test Data Set](#test-data-set)
-    - [Performance Validation](#performance-validation)
+  - [Performance Validation and Resource Requirements](#performance-validation)
```

Comment:
My concern here was that almost all users want to know what resources are required, but it was somewhat hidden under `Performance Validation`.

Reply:
Yeah i think this is a good change, more clearly outlines what they can expect to find